src/content/docs/integrations/open-source-telemetry-integrations/opentelemetry/view-your-opentelemetry-data-new-relic.mdx

```diff
@@ -13,7 +14,11 @@ After you import OpenTelemetry data into New Relic, you can use a variety of too
 
 ## Explorer: Get the big picture along with the details [#explorer]
 
-The New Relic **Explorer** tab is a good place to start gathering information about your services: 
+The New Relic **Explorer** tab is a good place to start gathering information about your services. It offers you a way to filter services and then provides a variety of views into your data.
+
+### Narrow down your data with filters [#narrow-your-search]
```

Comment:
```suggestion
### Narrow down your data with filters [#filter]
```

Reply:
@rkbodenner, I revamped this page, separating your data filtering information from the entity filtering information that was in the original. This anchor was changed in the process.